lib/LocationLookup/LocationModal.js

```diff
@@ -145,7 +145,11 @@ export default class LocationModal extends React.Component {
 
     if (!campusId) return;
 
-    const params = { query: `(campusId=="${campusId}")` };
+    const params = {
+      query: `(campusId=="${campusId}")`,
+      limit: '1000',
```

Comment:
it will be a bug again if number of libraries is more than 1K in particular campus...